src/components/ModelCamera/ModelCamera.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react'
+import dynamic from 'next/dynamic'
+import { Models } from "../../types/Models";
+import { loadModel } from './components/Model';
+
+
+const DynamicCamera = dynamic(() => import('./components/Camera'), {
+    ssr: false
+});
+
+const ModelCamera = () => {
+    const [models, setModels] = useState<Models>({
+        L_Model: undefined,
+        R_Model: undefined
+    })
+    const [isLoading, setIsLoading] = useState(true);
+    useEffect(() => {
+        // if (isLoading) {
```

Comment:
Why is this commented out? If it doesn't serve a purpose we should remove the comments.

Reply:
That is for loading the tensorflow model. That is the next step in the camera component